src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilTest.java

```diff
@@ -357,9 +357,9 @@ public void testGetPreviousSibling() {
     }
 
     @Test
-    public void testGetTokenNames() {
+    public void testGetLastTokenName() {
         assertEquals("Unexpected token name",
-            "OPTGROUP", JavadocUtil.getTokenName(10082));
+            "RP", JavadocUtil.getTokenName(10094));
```

Comment:
This test was redundant. Was previously equivalent to `JavadocUtilsTest#testGetTokenNameForId`. Hence modified accordingly.